code/datums/gamemode/role/cultist.dm

```diff
@@ -33,10 +33,10 @@
 	switch(greeting)
 		if ("roundstart")
 			to_chat(antag.current, {"<img src='data:image/png;base64,[icon2base64(logo)]' style='position: relative; top: 10;'/> <span class='sinister'><font size=3>You are a cultist of <span class='danger'><font size=3>Nar-Sie</font></span>!</font><br>
-				I, the Geometer of Blood want you to thin the veil between reality and his realm<br>
+				I, the Geometer of Blood, want you to thin the veil between reality and his realm<br>
```

Comment:
why is the geometer of blood talking about someone else's realm

Reply:
duh, I was tired probably